workers/unity/Packages/com.improbable.gdk.playerlifecycle/PlayerLifecycleHelper.cs

```diff
@@ -1,11 +1,13 @@
 using System;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
 using Improbable.Gdk.Core;
 using Improbable.PlayerLifecycle;
 using Unity.Entities;
 
 namespace Improbable.Gdk.PlayerLifecycle
 {
-    public static class PlayerLifecycleHelper
+    public class PlayerLifecycleHelper
```

Comment:
Why is this no longer static?

Reply:
ah was trying to debug something, forgot to change it back